blueprint/core/blueprint_CanvasView.h

```diff
@@ -394,6 +493,63 @@ namespace blueprint
                         return juce::var();
                     }
             });
+
+            // Text functions
+            setProperty("strokeText", juce::var::NativeFunction {
+                    [=](const juce::var::NativeFunctionArgs& args) -> juce::var {
+                        jassert(graphics);
+                        jassert(args.numArguments >= 3 && args.numArguments <= 4);
+
+                        const juce::String text = args.arguments[0].toString();
+                        const float        xPos = args.arguments[1];
+                        const float        yPos = args.arguments[2];
+
+                        // Default maxLineWidth to full context width
+                        float maxLineWidth = (float)image.getWidth();
```

Comment:
When we merge https://github.com/nick-thompson/blueprint/pull/56 we should ideally replace image.getWidth with ctxWidth.